src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -393,14 +397,18 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         parentId: string,
         permissions: BoxCommentPermission,
     }): void => {
-        const { file, api } = this.props;
+        const { api, emitAnnotationReplyDeleteEvent, file } = this.props;
 
+        emitAnnotationReplyDeleteEvent(id, parentId, true);
         api.getFeedAPI(false).deleteReply(
             file,
             id,
             parentId,
             permissions,
-            this.feedSuccessCallback,
+            () => {
```

Comment:
should we move all of these crud callbacks into separate functions instead of inlining in a call

Reply:
Makes sense. I've created separate onSuccess methods.